packages/curve-ui-kit/src/hooks/useMarketExtraIncentives.ts

```diff
@@ -0,0 +1,24 @@
+import { useMemo } from 'react'
+import { notFalsy } from '@curvefi/prices-api/objects.util'
+import type { MarketRateType, ExtraIncentive } from '@ui-kit/types/market'
+
+const CRV_ADDRESS = '0xd533a949740bb3306d119cc777fa900ba034cd52'
```

Comment:
we already have this is [multiple places](https://github.com/search?q=repo%3Acurvefi%2Fcurve-frontend%200xd533a949740bb3306d119cc777fa900ba034cd52&type=code)